### PR TITLE
Amplitude: Send event_id

### PR DESCRIPTION
**Pull request by ishachopra0401**
## Description of the change

Added a subheading for sending event_id 

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: docs/destinations/analytics/amplitude.mdx [hunk 1]
```diff
@@ -499,6 +499,26 @@ In order to use the Amplitude groups feature with RudderStack you need to define
 Even if you are not having an enterprise account or groups add-on, RudderStack would add `groups`as a `User Property` on a user's profile with `Group name trait` as its type and `Group value trait` as its value.
 
 Let' say you had defined `Group name trait` as `company_id` and `Group value trait` as `company_name` and made the above call then the user would be associated with a Group name : `RS` and a Group Value : `RudderStack` and you can also see that a user property with type `RS` and value `RudderStack` on user's profile.
+  
+## Send `event_id` in device mode
+
+RudderStack supports sending `event_id` to Amplitude in the device mode. It can be added under the `integrations` object and is supported for all the above mentioned API calls, namely, `identify`, `track`, `page`, `screen`, and `group`.
```
Comment (ameypv-rudder) [inline docs/destinations/analytics/amplitude.mdx:505]:
```suggestion
RudderStack supports sending `event_id` to Amplitude in the device mode. You can include it under the `integrations` object and it is supported for all the above-mentioned API calls, namely, `identify`, `track`, `page`, `screen`, and `group`.
```

File: docs/destinations/analytics/amplitude.mdx [hunk 2]
```diff
@@ -499,6 +499,26 @@ In order to use the Amplitude groups feature with RudderStack you need to define
...
+  
+A sample `identify` call with `event_id` looks like the following:
```
Comment (ameypv-rudder) [inline docs/destinations/analytics/amplitude.mdx:507]:
```suggestion
A sample `identify` call with `event_id` is shown below:
```

File: docs/destinations/analytics/amplitude.mdx [hunk 3]
```diff
@@ -499,6 +499,26 @@ In order to use the Amplitude groups feature with RudderStack you need to define
...
+
+```javascript
+rudderanalytics.identify(
+  "1hKOmRA4el9Zt1WSfVJIVo4GRlm", {
+    name: "Alex Keener"
+  }, {
+    integrations: {
+      Amplitude: {
+        event_id: 1234
```
Comment (ameypv-rudder) [inline docs/destinations/analytics/amplitude.mdx:516]: @ItsSudip is `event_id` a string or an int? if so, should it be presented within quotes, i.e. `"1234"`? cc @ishachopra0401

**Review by ameypv-rudder — APPROVED** [review]
LGTM - please proceed once the queries/suggestions are addressed.

File: docs/destinations/analytics/amplitude.mdx (hunk 3 above)
Comment (ItsSudip) [reply docs/destinations/analytics/amplitude.mdx:516]: It is an int. No need to provide quotations (").

**Review by ItsSudip — APPROVED** [review]

**Pull request merged**